### ISSUE TITLE
Failed to load file 'count3s.wav'.

### ISSUE THREAD

**Issue by kisik21** [itdelatrisu/opsu#153]
**Version:** 0.12.1
**Build date:** 2015-10-02 03:05
**OS:** Linux (amd64)
**JRE:** 1.7.0_85
**Error:** `Failed to load file 'count3s.wav'.`
**Stack trace:**

```
java.lang.IllegalArgumentException: Invalid format
    at org.classpath.icedtea.pulseaudio.PulseAudioDataLine.createStream(PulseAudioDataLine.java:142)
    at org.classpath.icedtea.pulseaudio.PulseAudioDataLine.open(PulseAudioDataLine.java:99)
    at org.classpath.icedtea.pulseaudio.PulseAudioDataLine.open(PulseAudioDataLine.java:283)
    at org.classpath.icedtea.pulseaudio.PulseAudioClip.open(PulseAudioClip.java:402)
    at itdelatrisu.opsu.audio.MultiClip.getClip(MultiClip.java:171)
    at itdelatrisu.opsu.audio.MultiClip.<init>(MultiClip.java:97)
    at itdelatrisu.opsu.audio.SoundController.loadClip(SoundController.java:120)
    at itdelatrisu.opsu.audio.SoundController.loadClip(SoundController.java:93)
    at itdelatrisu.opsu.audio.SoundController.init(SoundController.java:213)
    at itdelatrisu.opsu.states.Splash$2.run(Splash.java:157)
```

---

I have some problems with loading .wav files within opsu! client. What package I need to fix that bunch of almost same errors? (BTW, Ubuntu 14.04 Trusty)

**Comment by itdelatrisu** [comment]
Is this the same issue as #89?

**Comment by kisik21** [comment]
Yes, it's the same. I heard that I need to install Oracle Java to fix this. I installed it (I think), but did not tested it - I was sleepy. So I turned off the computer and decided to check today.

**Comment by kisik21** [comment]
com.sun.media.sound.DirectAudioDevice$DirectDL.implOpen(DirectAudioDevice.java:513)
	at com.sun.media.sound.DirectAudioDevice$DirectClip.implOpen(DirectAudioDevice.java:1304)
	at com.sun.media.sound.AbstractDataLine.open(AbstractDataLine.java:121)
	at com.sun.media.sound.DirectAudioDevice$DirectClip.open(DirectAudioDevice.java:1085)
	at com.sun.media.sound.DirectAudioDevice$DirectClip.open(DirectAudioDevice.java:1052)
	at itdelatrisu.opsu.audio.MultiClip.getClip(MultiClip.java:171)
	at itdelatrisu.opsu.audio.MultiClip.<init>(MultiClip.java:97)
	at itdelatrisu.opsu.audio.SoundController.loadClip(SoundController.java:120)
	at itdelatrisu.opsu.audio.SoundController.loadClip(SoundController.java:93)
	at itdelatrisu.opsu.audio.SoundController.init(SoundController.java:232)
	at itdelatrisu.opsu.states.Splash$2.run(Splash.java:157)
-----
So, I installed Oracle Java, but this error is given to me...

**Comment by itdelatrisu** [comment]
This probably got fixed with #89 in 039dd66, but feel free to re-open if it hasn't.
